.jenkins

```diff
@@ -193,6 +193,7 @@ pipeline {
                                   -D CMAKE_BUILD_TYPE=Debug \
                                   -D CMAKE_CXX_COMPILER_LAUNCHER=ccache \
                                   -D CMAKE_CXX_COMPILER=pgc++ \
+                                  -D CMAKE_CXX_FLAGS="-Wall" \
```

Comment:
Where did you find that flag? It's not in the [documentation](https://www.pgroup.com/resources/docs/19.4/x86/pgi-user-guide/index.htm)

Reply:
I was removing everything that didn't work. Not sure if it has any effect. At least the compiler output didn't change. Replaced now with `-Minform=warn`. I don't see anything either, but at least it is documented (https://www.pgroup.com/resources/docs/19.4/x86/pgi-ref-guide/index.htm#cmdln-m-opts-misc__Minform).